server/services/version.ts

```diff
@@ -34,6 +35,7 @@ export async function filterVersion<T>(user: UserDoc, unfiltered: T): Promise<T>
 export async function findVersionById(user: UserDoc, id: ModelId, opts?: GetVersionOptions) {
   let version = VersionModel.findById(id)
   if (opts?.thin) version = version.select({ state: 0, logs: 0, metadata: 0 })
+  if (!opts?.showLogs) version = version.select({ logs: 0 })
```

Comment:
Would there be an issue if `opts.thin === true` and `opts.showLogs === false`? Would line 38 overwrite the data fetched on line 37?

Reply:
I did think about this, and I actually think there is somewhere this could happen, so I'm going to change it back.